.github/ISSUE_TEMPLATE/presentation.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
This template is also introduced in this PR: https://github.com/cncf/tag-env-sustainability/pull/10/files

Reply:
Yes because the gov was referring to it. Would use the templates here, to have a clean cut.